LeaderboardBackend.Test/Services/AuthServiceTests.cs

```diff
@@ -0,0 +1,58 @@
+using NUnit.Framework;
+using Moq;
+using LeaderboardBackend.Services;
+using LeaderboardBackend.Models;
+using Microsoft.Extensions.Configuration;
+using System.Text;
+using System.IdentityModel.Tokens.Jwt;
+using Microsoft.IdentityModel.Tokens;
+
+namespace LeaderboardBackend.Test.Services;
+
+public class AuthServiceTests
+{
+	[Test]
+	public void GenerateJSONWebToken_CreatesTokenWithConfig()
+	{
+		string key = "testkeythatsatisfiesthecharacterminimum";
+		string issuer = "leaderboards.gg";
+
+		string configJson = string.Format(@"
+		{{
+			""Jwt"": {{
+				""Key"": ""{0}"",
+				""Issuer"": ""{1}""
+			}}
+		}}
```

Comment:
We could try making the keys camelCase here maybe?

Reply:
I'm matching it exactly to how `appsettings.json` does it, and in the AuthService they are accessed with their pascal cased names.

Reply:
Ah, okay then.